README.md

```diff
@@ -0,0 +1,21 @@
+# `mkctr`: cross platform container builder for go
+
+`mkctr` is a small go binary which uses `GOOS= GOARCH= go build` directly to compile go binaries and then uses [go-containerregistry](https://github.com/google/go-containerregistry) to create and publish the new containers based on the desired platforms.
+
+This is inspired by [ko](https://github.com/google/ko) which is awesome but doesn't support multiple binaries in a single container.
+
+## Usage
+
+```bash
+mkctr \
```

Comment:
where do you specify the GOOS/GOARCH values? show that?

Reply:
they are auto discovered based on the provided image, added a note.